apps/widget/cypress/support/commands.ts

```diff
@@ -13,51 +14,58 @@ Cypress.Commands.add('openWidget', (settings = {}) => {
   return cy.get('#notification-bell').click();
 });
 
-Cypress.Commands.add('initializeShellSession', (subscriberId, identifier, encryptedHmacHash) => {
-  cy.visit('http://localhost:4700/cypress/test-shell', { log: false });
-
-  return cy
-    .window()
-    .then((w) => {
-      const subscriber = {
-        subscriberId: subscriberId,
-        firstName: faker.name.firstName(),
-        lastName: faker.name.lastName(),
-        email: faker.internet.email(),
-        subscriberHash: encryptedHmacHash,
-      };
-
-      w.novu.init(
-        identifier,
-        {
-          unseenBadgeSelector: '#unseen-badge-span',
-          bellSelector: '#notification-bell',
-        },
-        subscriber
-      );
-
-      return subscriber;
-    })
-    .then(function (subscriber) {
-      return cy
-        .get('#novu-iframe-element')
-        .its('0.contentDocument.body')
-        .should('not.be.empty')
-        .then((body) => {
-          return cy
-            .wrap(body)
-            .window()
-            .then((w) => w.localStorage.clear());
-        })
-        .then(function () {
-          return subscriber;
-        });
-    });
-});
+Cypress.Commands.add(
+  'initializeShellSession',
+  ({ subscriberId, identifier, encryptedHmacHash, tabs, stores }: IInitializeShellSessionSettings) => {
+    // eslint-disable-next-line no-console
+    console.log('tabs ', tabs);
```

Comment:
On purpose or a leftover (🗑️ )?

Reply:
Nice catch, Thanks!!! :) 
I threw the leftovers in the trash 👍